src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -233,12 +253,11 @@ void CustomTextRenderer::_FillRectangle(void* clientDrawingContext,
 
     // Since we've delegated the drawing of the background of the text into this function, the origin passed in isn't actually the baseline.
     // It's the top left corner. Save that off first.
-    D2D1_POINT_2F origin = D2D1::Point2F(baselineOriginX, baselineOriginY);
+    const D2D1_POINT_2F origin = D2D1::Point2F(baselineOriginX, baselineOriginY);
 
     // Then make a copy for the baseline origin (which is part way down the left side of the text, not the top or bottom).
     // We'll use this baseline Origin for drawing the actual text.
-    D2D1_POINT_2F baselineOrigin = origin;
-    baselineOrigin.y += drawingContext->spacing.baseline;
+    const D2D1_POINT_2F baselineOrigin = { origin.x, origin.y + drawingContext->spacing.baseline };

```

Comment:
prefer universal initialization over assignment
```suggestion
    const D2D1_POINT_2F baselineOrigin{ origin.x, origin.y + drawingContext->spacing.baseline };
```

Reply:
done